test/functionalTest/cases/2763_postUpdateContext_error_return/postEntities_with_errors_in_service_path.test

```diff
@@ -0,0 +1,77 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+postEntities with errors in service-path
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Try to create an entity with more than one service path - see error
+# 02. Make sure the entity doesn't exist in the database
+#
+
+echo "01. Try to create an entity with more than one service path - see error"
+echo "======================================================================="
+payload='{ "id": "E1", "type": "T1" }'
+orionCurl --url /v2/entities --payload "$payload" --servicePath /E1,/E2
+echo
+echo
+
+
+echo "02. Make sure the entity doesn't exist in the database"
+echo "======================================================"
+echo "db.entities.findOne()" | mongo ftest
```

Comment:
Why don't use the GET /v2/entities operation to check it? We should use keep direct mongo queries only as last resort measure when the API doesn't provide alternatives.

(Btw, I think we have a mongoCmd() function for test harness to avoid direct calls to mongo shell)